api/wasm-interpreter/Cargo.toml

```diff
@@ -16,7 +16,7 @@ homepage = "https://slint-ui.com"
 crate-type = ["cdylib"]
 
 [dependencies]
-slint-interpreter = { path = "../../internal/interpreter", default-features = false, features = ["std", "backend-winit", "renderer-winit-femtovg", "compat-0-3-0"] }
+slint-interpreter = { path = "../../internal/interpreter", default-features = false, features = ["std", "highlight", "backend-winit", "renderer-winit-femtovg", "compat-0-3-0"] }
```

Comment:
Should this be a optional feature of the interpreter.  We don't need to build that for the interpreter that is used for the documentation and other website integration.